charts/rpe-feature-toggle-api/templates/importer.yaml

```diff
@@ -36,24 +37,33 @@ spec:
               resourcegroup: {{ include "importer.resourcegroup" . | quote }}
               keyvaultobjectnames: "admin-username-cmc;admin-password-cmc"
               keyvaultobjecttypes: "secret;secret" # OPTIONS: secret, key, cert
+      {{- end }}
       containers:
         - name: {{ template "hmcts.releaseName" . }}-importer
           image: {{ .Values.importer.oneoff.image }} 
           imagePullPolicy: {{ .Values.importer.oneoff.imagePullPolicy }}
           command: ["sh", "-c", "{{ .Values.importer.oneoff.command }}"]
+          {{- if .Values.importer.kvSecretRef }}
           volumeMounts:
           - name: kvcreds
             mountPath: /kvmnt
             readOnly: true
+          {{- end }}
           env:
+          {{- if .Values.importer.kvSecretRef }}
           - name: IMPORTER_CREDS_MOUNT
             value: "/kvmnt"
+          {{- end }}
           - name: FEATURE_TOGGLE_API_URL
-            value: "http://{{ template "hmcts.releaseName" . }}/api/ff4j/store/features/"
+            value: "{{ .Values.importer.apiUrl }}"
           - name: PERMISSIONS
             value: "{{ .Values.importer.oneoff.permissions }}"
           - name: VERBOSE
             value: "true"
+          - name: ADMIN_USERNAME
```

Comment:
it may take priority but it would be good to add guards around as its confusing if someone describes the pod and sees these values but they aren't actually being used

Reply:
good point 👍 will sort